broker-daemon/block-order-worker/index.js

```diff
@@ -402,55 +416,69 @@ class BlockOrderWorker extends EventEmitter {
       throw new Error(`No engine available for ${counterSymbol}`)
     }
 
-    const promisedFills = orders.map(async (order) => {
+    const ordersByOrderId = await Promise.all(orders.map((order) => {
```

Comment:
this is a bit of a confusing name for what this variable holds, especially since it's the same name as the index